mozdownload/scraper.py

```diff
@@ -266,12 +266,14 @@ def total_seconds(td):
         self.logger.info('Downloading from: %s' % self.url)
         self.logger.info('Saving as: %s' % self.filename)
 
+        if print_url:
+            print self.url
```

Comment:
Please see the line 266 above. We already print the final URL here. The issue itself is that we want to print the URL without downloading the build. Sorry if that was not that clear in the description.